src/Simulator/IncrementalRedstoneSimulator/ObserverHandler.h

```diff
@@ -0,0 +1,128 @@
+
+#pragma once
+
+#include "RedstoneHandler.h"
+
+
+
+
+
+class cObserverHandler : public cRedstoneHandler
+{
+public:
+
+	inline static bool IsOn(NIBBLETYPE a_Meta)
+	{
+		return (a_Meta & 0x8) == 0x8;
+	}
+
+	inline static Vector3i Getಠ_ಠOffset(NIBBLETYPE a_Meta)
```

Comment:
Not everyone has ಠ keys on their keyboard, maybe this could be made something ASCII-compatible?

Reply:
ಠ‿↼

Reply:
I am seriously appalled by the fact that this is legal C++. I definitely vote we don't use any of the extended ASCII characters *at all*.
I've already got a CheckBasicStyle.lua filter for this, there are exactly 4 violations in the code right now, all of them in comments. Two of them are clear mistakes, two are somewhat legitimate (`§' in ChatColor.cpp), but still not exactly needed.

Reply:
![Hello World](https://user-images.githubusercontent.com/2939130/79044345-7b5eb200-7bfc-11ea-8246-c62d65e5fafb.png)